### ISSUE TITLE
1.6-beta.26: React component error

### ISSUE THREAD

**Issue by devarthurribeiro** [meteor/meteor#9091]
when upgrading from version 1.5.2 to 1.6-beta.26 I get the following error in almost all my compoents.
I'm taking more information. I'm close to figuring out what causes this error.

React.createElement: type is invalid -- expected a string (for built-in components) 

Packages
```meteor-base@1.1.0
mobile-experience@1.0.4
mongo@1.2.0
blaze-html-templates@1.0.4
reactive-var@1.0.11
jquery@1.11.10
tracker@1.1.3

standard-minifier-css@1.3.4
standard-minifier-js@2.1.1
es5-shim@4.6.15
ecmascript@0.8.2
shell-server@0.2.4

insecure@1.0.7
fourseven:scss
materialize:materialize@=0.98.2
accounts-password@1.4.0
accounts-google@1.2.0
accounts-facebook@1.2.1
accounts-base@1.3.2
react-meteor-data
momentjs:moment
kevohagan:sweetalert
crosswalk@1.7.1
react-template-helper
reywood:publish-composite
gadicc:blaze-react-component
okgrow:analytics
dynamic-import@0.1.1
service-configuration@1.0.11
apollo
force-ssl@1.0.14
lepozepo:s3

```

**Comment by devarthurribeiro** [comment]
Find out the reason for the error, caused due to my index.js file:

index.js
```
export * from './RecoveryPassword';
```
my import

```
import RecoveryPassword from '/imports/ui/components/RecoveryPassword/
```
solved with

```
import RecoveryPassword from '/imports/ui/components/RecoveryPassword/RecoveryPassword'
```

Reify not detect index.js file?

**Comment by benjamn** [comment]
This was a ECMAScript spec compliance change in Meteor 1.5.2 (mentioned in [`History.md`](https://github.com/meteor/meteor/blob/devel/History.md#v152-2017-09-05)):

* The `reify` npm package has been upgraded to version 0.12.0, which includes a minor breaking [change](https://github.com/benjamn/reify/commit/8defc645e556429283e0b522fd3afababf6525ea) that correctly skips exports named `default` in `export * from "module"` declarations. If you have any wrapper modules that re-export another module's exports using `export * from "./wrapped/module"`, and the wrapped module has a `default` export that you want to be included, you should now explicitly re-export `default` using a second declaration:
  ```js
  export * from "./wrapped/module";
  export { default } "./wrapped/module";
  ```
In other words, I would add
```js
export { default } from './RecoveryPassword';
```
to your `index.js` file.
